### ISSUE TITLE
Support for Ingress @ networking.k8s.io/v1

### ISSUE THREAD

**Issue by jkremser** [k8gb-io/k8gb#847]
Since k8s@1.22 the Ingress with version `extensions/v1beta1` nor `networking.k8s.io/v1beta1` are no longer supported. The only supported version is now `networking.k8s.io/v1`

If k8gb is deployed to a kubernetes cluster with version 1.22 or higher, it keeps crash-looping on this error:

```
2022-02-16T16:03:02Z ERR github.com/k8gb-io/k8gb/controllers/logging/logr.go:60 > gslb: if kind is a CRD, it should be installed before calling Start {"reconciler group":"k8gb.absa.oss","reconciler kind":"Gslb"} error="no matches for kind \"Ingress\" in version \"networking.k8s.io/v1beta1\""
```

**Comment by somaritane** [comment]
@jkremser, this is how it was done in [helm.sh](https://helm.sh/) for sample chart creation (with `helm create` command):
https://github.com/helm/helm/pull/9621/files#diff-92d0c93dff3a254f887d928f60f6b46ad1da1cbe7a5737dda4cd85cd72876092R10

**Comment by jkremser** [comment]
right, in theory we can handle this issue also on the helm chart level. Having those `(semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion))` and deploy older version of k8gb for older k8s clusters, but this smells like some auto-magic behavior and I am not a big fan of this approach, the less logic in helm, the better, imho

I was also looking into that possibility of having multiple versions of CRDs present at the same time (because this change also impacts the shape under `.spec.ingress` in our gslbs) which can be handy also in the future. There is a nice tutorial how to do that on [kubebuilder web](https://book.kubebuilder.io/multiversion-tutorial/tutorial.html). But it requires a new webhook, cert-manager and what not. I've actually started this approach and then abandoned it after writing those conversion methods.
